treasury/src/constants/constants.ts

```diff
@@ -37,6 +39,7 @@ export const minDeposit: { [vault in TreasuryVaultOptions]: BigNumber } = {
   "rSAMB-TSRY": ethers.utils.parseEther("3000"),
   "rVIP-wBTC": BigNumber.from(10).pow(getAssetDecimals(getAssets("rVIP-wBTC"))), // minDeposit of 1 wBTC
   "rVIP-USDC": BigNumber.from(10).pow(getAssetDecimals(getAssets("rVIP-USDC"))), // minDeposit of 1 USDC
+  "rVIP-VOL": BigNumber.from(0),
```

Comment:
Can change to 0.1 (BigNumber.from(10).pow(5)) if we want, that's what it's set at the contract level